src/components/SearchGrid.vue

```diff
@@ -4,16 +4,18 @@
                                :query="query"
                                :searchTerm="searchTerm"
                                @onLoadMoreImages="onLoadMoreImages" />
-    <search-grid-manual-load v-else-if="renderManualLoad"
+    <search-grid-manual-load v-else
```

Comment:
Could you revert to `v-else-if="renderManualLoad"` please?

Reply:
Sure. Let me make the changes.